requirements.txt

```diff
@@ -0,0 +1,3 @@
+flask
```

Comment:
I see there is an installs.txt file, not sure of the difference but requirements.txt seems to be the python standard from the examples i found